demo/src/app/case-view-consumer.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'case-create-consumer',
+  template: `<div class="container-fluid">
+                <ccd-case-view [case]="caseId"></ccd-case-view>
+             </div>`
+})
+export class CaseViewConsumerComponent implements OnInit {
```

Comment:
should the name be CaseCreateConsumerComponent?

Reply:
I think this name is actually correct as it is consistent with the other consumers like CaseCreateConsumerComponent or CaseProgressConsumerComponent.